src/views/Login.jsx

```diff
@@ -11,6 +13,14 @@ export class Login extends React.Component {
     }
   }
 
+  componentWillReceiveProps (nextProps, nextContext) {
+    // if token and user exists, then we're already logged in so redirect
+    // TODO: validate token
+    if (nextProps.user && nextProps.token) {
+      // TODO redirect
```

Comment:
so under the hood react-router uses https://github.com/mjackson/history, and since Login is a direct decendent of ther router, it is available w/ `this.props.history`. You can check their docs for the available methods, https://github.com/mjackson/history/blob/master/docs/GettingStarted.md, but you probably want `history.push` or `history.replace` depending on if you want be able to go back to that page, or replace it completely.